### PR TITLE
Store update to bring it inline with the SF store

### PR DESCRIPTION
**Pull request by Lukewh**

### CODE REVIEW

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7272/

File: jujugui/static/gui/src/app/components/store/test-store.js [hunk 1]
```diff
@@ -43,18 +43,20 @@ describe('Store', function() {
     assert.equal(output.props.children[0].props.children.length, 3);
   }),
 
-  it('can render big data feature in the correct place', function() {
+  fit('can render big data feature in the correct place', function() {
```
Comment (frankban) [inline jujugui/static/gui/src/app/components/store/test-store.js]: fit

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7276/

**Comment by Lukewh** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**